qa/packaging/src/test/java/org/elasticsearch/packaging/util/docker/DockerRun.java

```diff
@@ -112,6 +115,13 @@ String build() {
         // Limit container memory
         cmd.add("--memory " + memory);
 
+        // Add default java opts
+        if (this.envVars.containsKey("CLI_JAVA_OPTS")) {
+            this.envVars.put("CLI_JAVA_OPTS", this.envVars.get("CLI_JAVA_OPTS") + " " + DEFAULT_JAVA_OPTS);
```

Comment:
Should we use this with ES_JAVA_OPTS as well?

Reply:
> Should we use this with ES_JAVA_OPTS as well?

We _could_. It doesn't affect this particular issue as security auto config is done in the CLI.

Reply:
I went ahead and added it on the chance we add anything else to `DEFAULT_JAVA_OPTS`.